src/vsphere_cpi/spec/integration/vm_existing_disk.rb

```diff
@@ -0,0 +1,84 @@
+require 'integration/spec_helper'
+
+describe 'Creating VM with existing persistent disks' do
+  before (:all) do
+    @datacenter_name = fetch_and_verify_datacenter('BOSH_VSPHERE_CPI_DATACENTER')
+    @cluster_name = fetch_and_verify_cluster('BOSH_VSPHERE_CPI_CLUSTER')
+    @datastore_shared_pattern = fetch_and_verify_datastore('BOSH_VSPHERE_CPI_SHARED_DATASTORE', @cluster_name)
+    @cluster_one_datastore = fetch_and_verify_datastore('BOSH_VSPHERE_CPI_DATASTORE_PATTERN', @cluster_name)
+  end
+
+  context 'when there are existing persistent disks ' do
```

Comment:
Creating VM with existing persistent disks 'when disks needs to be migrated to new datastore ' cpi should be able to create a VM. I think context needs to be updated to make clear what you are actually testing